spec/models/article_spec.rb

```diff
@@ -0,0 +1,38 @@
+require 'rails_helper'
+
+RSpec.describe Article, type: :model do
+  
+  it 'should have valid Factory' do
+    expect(create(:article)).to be_valid
+  end
+
+  describe 'Database table' do
+    it { is_expected.to have_db_column :title }
+    it { is_expected.to have_db_column :teaser }
+    it { is_expected.to have_db_column :content }
+  end
+
+  describe 'Validations' do
+    it { is_expected.to validate_presence_of :title}
+    it { is_expected.to validate_presence_of :teaser }
+    it { is_expected.to validate_presence_of :content } 
+  end
+
+  describe "Check if new article is created" do
```

Comment:
johan and I are wondering if this test should be in the other spec file?

Reply:
done